src/parse-query.js

```diff
@@ -0,0 +1,2 @@
+import words from 'lodash/words';
+export const parseQuery = ({ ft } = {}) => words(ft, /[^,]+/g);
```

Comment:
regex and `words` is a bit overkill here, lets change this to [split](https://lodash.com/docs/4.17.10#split) instead.

Reply:
Ok. I thought it might be.